src/status_im/contexts/onboarding/common/carousel/view.cljs

```diff
@@ -65,7 +64,7 @@
           nil)]
     [rn/view {:style {:flex 1}}
      [gesture/gesture-detector {:gesture identified-gesture}
-      [container-view {:style (style/carousel-container carousel-left animate? (* 4 window-width))}
+      [container-view {:style (style/carousel-container carousel-left animate? (* 3 window-width))}
```

Comment:
Suggestion: extract the number of pages `3` as a constant and reuse it.

Reply:
Yes, I will address this while refactoring carousel related code.